website/pages/en/network/developing.mdx

```diff
@@ -2,52 +2,122 @@
 title: Developing
 ---
 
-Developers are the demand side of The Graph ecosystem. Developers build subgraphs and publish them to The Graph Network. Then, they query live subgraphs with GraphQL in order to power their applications.
+Learn developer fundamentals and review the basics of a subgraph’s lifecycle. To start coding right away and engage The Graph, go to the following: [Developer Quick Start](/quick-start/)
 
-## Subgraph Lifecycle
+## Overview
 
-Subgraphs deployed to the network have a defined lifecycle.
+As a developer, you need important data to build and power your dapps. Querying and indexing that blockchain data can be challenging, but The Graph provides a solution to this issue.
 
-### Build locally
+On The Graph, you can:
 
-As with all subgraph development, it starts with local development and testing. Developers can use the same local setup whether they are building for The Graph Network or a local Graph Node, leveraging `graph-cli` and `graph-ts` to build their subgraph. Developers are encouraged to use tools such as [Matchstick](https://github.com/LimeChain/matchstick) for unit testing to improve the robustness of their subgraphs.
+1.  Create, deploy and publish subgraphs to The Graph using **Graph CLI** and [Subgraph Studio](/https://thegraph.com/studio/).
+2.  Use Graph node via **GraphQL** to query live subgraphs and power dapps.
```

Comment:
```suggestion
2.  Use Graph Node via **GraphQL** to query live subgraphs and power dapps.
```
or `graph-node`? But looks like lots of pages call it "Graph Node" so I guess that's fine.

Reply:
Wait also, kinda related to my point below: why are we referencing Graph Node here? I don't think subgraph developers need to know about it; they don't interact with Graph Node at all. They make GraphQL queries to an API endpoint which returns them data.